### DISCUSSION TITLE
IR Sensor Angles and Values?

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#478] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Mac OS, Fusion VM, Ubuntu 20.04

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

G.5.3

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

Yes: 2.4GHz Home WiFi 

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

Yes

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

Is there a spec for the IR Sensors - angles and formula for distance from value?

(This [page](https://iroboteducation.github.io/create3_docs/api/hazards/) has a 404 link to [the ir intensity sensor](https://iroboteducation.github.io/hw/mechanical/#ir-proximity-sensors) info.  [Found it: ](https://iroboteducation.github.io/create3_docs/hw/mechanical/#ir-proximity-sensors)

This is my empirical result: (I think I got 0 deg opposite for a right-hand - should be left 0, no?)
![IR_Sensor_Angles](https://github.com/iRobotEducation/create3_docs/assets/8950202/cc3da01b-aee8-44e7-8226-94ad62ab0e8e)

Anyone write a point-cloud rebroadcaster with the 7 ir_intensity sensor distances (with beam width, min/max distance, resolution, ...)?

**Comment by shamlian** [comment, marked as answer]
Thanks for reporting the bug in the docs in passing. The IR sensors are obstacle sensors which report the intensity of reflected infrared light (subtracting out the ambient), and therefore do not report albedo-independent distance. I can put in a PR to fix the broken link.
